packages/components/managed-data-table/src/data-table-settings/droppable-panel/messages.js

```diff
@@ -0,0 +1,10 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  useInputSearch: {
+    id: 'UIKit.ManagedDataTable.ColumnManager.DroppablePanel.useInputSearch',
+    description:
+      'A message to indicate to the user that he can load more attributes by using the search input',
```

Comment:
Let's use a neutral gender.

```suggestion
      'Indicate that the user can use the search input to load more attributes',
```

Reply:
I'm pinging translations to see if I can get this new wording translated.